dnf/cli/commands/install.py

```diff
@@ -74,15 +74,18 @@ def run(self, extcmds):
                 done = True
 
         # Install packages.
+        errs = []
         for pkg_spec in pkg_specs:
             try:
                 self.base.install(pkg_spec)
             except dnf.exceptions.MarkingError:
```

Comment:
Hm, I wouldn't reraise the exception (aka hold it in a variable) since it may contain the `pkg_spec` (see https://github.com/rpm-software-management/dnf/blob/master/dnf/base.py#L1335) and it would be confusing that it's just one of them. Better raise a new one. But I agree with the rest.